src/rgw/rgw_file.h

```diff
@@ -177,7 +177,7 @@ namespace rgw {
 
   class RGWFileHandle : public cohort::lru::Object
   {
-    struct rgw_file_handle fh;
+    struct rgw_file_handle fh{};
```

Comment:
I don't see a reason to introduce this syntax, which is not part of the prevailing style.  If you believe fh can ever be uninitialized, please initialize it in the applicable constructors.  Note that objects of this type are required to be created by a factory method, which also carries weight here.

Reply:
@mattbenjamin What you suggested is my preferred method too - initialize those variable in the constructors. But I saw that many of these structures (initialized with `{}`) are having structure / enum variables inside. So those variables should be default initialized in their own constructors too. So that is the reason I thought `{}` would be simple. 

```
struct rgw_file_handle
{
  struct rgw_fh_hk fh_hk;
  void *fh_private;
  enum rgw_fh_type fh_type;
};
```

Reply:
right, it's a C structure

Reply:
@mattbenjamin Agreed. `fh_hk` initialization is missing in `RGWFileHandle(RGWLibFS* _fs) {}`. The other places seems to be okay.

Reply:
@amitkumar50 can you please update?

Reply:
@mattbenjamin @joscollin Working on it. Thanks For comment